saltgui/static/scripts/routes/job.js

```diff
@@ -26,7 +26,19 @@ class JobRoute extends Route {
     });
 
     const container = this.getPageElement().querySelector(".job-info");
-    container.querySelector('.function').innerHTML = info.Function;
+
+    let functionText = info.Function + " on ";
+    if(info["Target-type"] != "glob" && info["Target-type"] != "list") {
```

Comment:
GLOB is the default case. This would lead to `glob *` or `glob hostname` in most cases. When it is from a `salt` commandline, also no explicit targettype would be used there. Showing the information without the `glob` keyword is therefore less confusing.

LIST has a very readable format already. e.g. `hostname1,hostname2,hostname3`. No real need to add the `list`keyword. If that were the only reason, I would not have made the exception. But due to a bug in 2018.3, most historic jobs are returned as is they were of type `list`. That caused the balance to tip towards not showing the `list` keyword.